src/init/init.ts

```diff
@@ -43,14 +43,17 @@ const templateDescriptions: {[name: string]: string} = {
   'shop': 'The "Shop" Progressive Web App demo',
 };
 
+// TODO: Add Shop "^2.0.0" generator once Polymer 2.0 Shop template is released.
 let shopGenerator = createGithubGenerator({
   owner: 'Polymer',
   repo: 'shop',
+  semverRange: '^1.0.0',
 });
-
+// TODO: Add Shop "^3.0.0" generator once Polymer 2.0 PSK template is released.
```

Comment:
So with this we have to update the code if they ship a new major version. Ok for now, but we might want to make templates self-serving, where we show the last 2 majors or something.